src/instructlab/model/train.py

```diff
@@ -246,6 +252,7 @@
 @click.option(
     "--nnodes",
     type=int,
+    default=1,
```

Comment:
Why are you adding defaults? All defaults should come from `config.yaml` via Click's `default_map`. Please remove all defaults that are coming from config.

Reply:
https://github.com/instructlab/instructlab/pull/1763 is a bug fix for a debug feature that will help you to figure out the source of a default. I left the corrected output in a comment.